### ISSUE TITLE
Compare with Vaswani et al. (2017) on EN2DE in "Scaling Neural Machine Translation"

### ISSUE THREAD

**Issue by xiaoda99** [pytorch/fairseq#345]
Dear guys,

As shown in Table 1 of "Scaling Neural Machine Translation" paper, your reference reimplementation of transformer-big on EN2DE reach the same valid BLEU as Vaswani et al. (2017) with much fewer updates (300k -> 192k). Considering that your reimplementation basically replicates all hparam settings of Vaswani et al. (2017), can you explain what cause it to converge faster?

- Da Xiao

**Comment by myleott** [comment]
In Table 1 we stop as soon as the valid loss is below 2.11, which happens after ~192K updates (which is an average across three runs). I believe if you continue training to 300K updates with the smaller batch size (25K tokens) then it mostly plateaus. It seems possible that Vaswani et al. also converged after ~192K updates but used 300K across their runs to ensure convergence.
